cli/internal/context/context.go

```diff
@@ -28,6 +28,7 @@ const GLOBAL_CACHE_KEY = "snozzberries"
 
 // Context of the CLI
 type Context struct {
+	RootPackageInfo  *fs.PackageJSON // TODO(gsoltis): should this be included in PackageInfos?
```

Comment:
I think eventually this will be. Big feature request is the ability to be able to run tasks from root. I wonder if this is as easy as just adding root to to package infos. and then removing the special resolution logic, but keeping the special edge connecting logic? 

We also will need `inputs:[]` or something to restrict what should be hashed, but I'll spec that.